src/coreComponents/physicsSolvers/fluidFlow/wells/SinglePhaseWell.cpp

```diff
@@ -286,7 +288,7 @@ void SinglePhaseWell::updateFluidModel( WellElementSubRegion & subRegion, localI
   } );
 }
 
-void SinglePhaseWell::updateSubRegionState( WellElementSubRegion & subRegion, localIndex const targetIndex )
+void SinglePhaseWell::updateSubRegionState( MeshLevel const & meshLevel, WellElementSubRegion & subRegion, localIndex const targetIndex )
```

Comment:
passing the `MeshLevel` and a `Subregion` isn't really what we want to be doing since one contains the other.